edx-platform/pearson-kaust-theme/lms/templates/courseware/courses.html

```diff
@@ -46,42 +46,3 @@ <h1>${_("Find your course")}</h1>
 
   </div>
 </section>
-
-<main id="main" aria-label="Content" tabindex="-1" class="hidden">
```

Comment:
Why was this block removed? @amalbas

Reply:
@Squirrel18 the content inside this block is not used in prod, and is removed in juniper (journal part)

Reply:
Ok, thanks. @amalbas